gpd/pocket-4/default.nix

```diff
@@ -11,6 +11,9 @@ in
     ../../common/hidpi.nix
   ];
 
+  # Needed for wifi
+  hardware.enableRedistributableFirmware = mkDefault true;
```

Comment:
Can we be specific about the firmware we need? this is just a wildcard for everything we have in nixos.